src/place/fields.js

```diff
@@ -0,0 +1,466 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    AutocompleteInput,
+    CompoundInput,
+    DateInput,
+    TextInput,
+    TermPickerInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  return {
+    document: {
```

Comment:
This shouldn't have the `document` level, so that consumers of this plugin can include the 'ns2:places_nagpra' node inside their `document` config using a spread operator. You want to use the `collectionobject` and `claim` directories in this project as examples.